app/views/articles/_sidebar_nav.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="sidebar-nav" id="sidebar-nav">
   <% if user_signed_in? %>
-    <div class="sidebar-profile-snapshot">
+    <div class="sidebar-profile-snapshot" style="display: none;"> <%# TODO: i'd rather get rid of this whole section, but removing it actually breaks some JS :( %>
```

Comment:
I think in the spirit of frog boiling, we might want to keep this around and do a full removal as a future step.

Reply:
Good point, I'll bring it back.

Reply:
Consider it done.